interfaces/builtin/network_manager.go

```diff
@@ -88,6 +88,8 @@ network packet,
 
 /run/udev/data/* r,
 
+/etc/gnutls/config r,
```

Comment:
I am open for suggestions around whether this should be located in a more generic interface